content/docs/command-reference/exp/show.md

```diff
@@ -127,6 +129,14 @@ metric or param.
 
 - `--md` - prints the command's output in Markdown table format.
 
+- `--html` - generates an interactive parallel coordinates plot from the table.
```

Comment:
Can we link somewhere that explains what a PCP is perhaps? Or even better, briefly explain it here... In fact ideally we should do so as a new passage in the **Description** (that mentions this option).

Reply:
p.s. unless we split this feature to a separate command per https://github.com/iterative/dvc/pull/6933#issuecomment-973159973

Reply:
If we keep this feature here (included in `exp show`) we need to redefine the entire `exp show` description. It's no longer just "prints a customizable table of experiments, their metrics and parameters." PCP is a plot (not a table) and only includes metrics right? Or can it also include params

Reply:
The PCP contains the same data that it's in the columns of the table. To reduce noise, we always apply a filter to drop columns that have the same value for all rows. But if a `param` has different values per row, it will be shown in the plot.

Reply:
Right, right. It has metrics and params (some filtered) I was confused. But still, the description of `exp show` needs to be a bit more general if we keep this feature here. Currently it's "prints a customizable table of experiments..." Maybe "displays your experiments in a customizable table or parallel coordinates plot"?

Reply:
> p.s. sorry for the delay.